### PR TITLE
added Makefile and some test stuff for cbmcvt

### PR DESCRIPTION
**Pull request by mrdudz**
This PR is not (yet) ment for actual merging, but for discussion and review, see issue #1100

### CODE REVIEW

File: util/cbm/Makefile [hunk 1]
```diff
@@ -0,0 +1,100 @@
+.SILENT:
+
+PETCAT ?= petcat
+
+CL65 ?= cl65
+
+DEL = $(RM)
```
Comment (polluks) [inline util/cbm/Makefile:7]: Why?

**Comment by oliverschmidt** [comment]
So is it okay for everybody to nuke cbmcvt?

**Comment by oliverschmidt** [comment]
Thanks for your effort anyhow.

**Comment by mrdudz** [comment]
Deleted code is best code, zero bugs :)

**Comment by oliverschmidt** [comment]
:-))

**Pull request closed without merging**